code/datums/mind.dm

```diff
@@ -185,7 +185,7 @@
 	if(!recipient)
 		recipient = current
 	var/list/output = list()
-	output.Add("<B>[current.real_name]'s Memories:</B><HR>")
+	output.Add("<meta charset='UTF-8'><b>[current.real_name]'s Memories:</b><hr>")
```

Comment:
Are all these charset specifications required? HTML5 defaults to `UTF-8` https://www.w3schools.com/charsets/ref_html_utf8.asp
Having said that, I know BYOND is often unexpected.

Reply:
> Are all these charset specifications required? HTML5 defaults to `UTF-8` https://www.w3schools.com/charsets/ref_html_utf8.asp Having said that, I know BYOND is often unexpected.

Yes, excatly that - we need to specify for byond. You can easily check it by using any cyrillic in htmls

Example is https://github.com/ss220club/Paradise-SS220/pull/1014 , where specifying is actually fixing stuff